rawx-apache2/src/rawx_repository.c

```diff
@@ -1033,6 +1034,54 @@ dav_rawx_move_resource(dav_resource *src_res, dav_resource *dst_res,
 	return e;
 }
 
+static GError *
+remove_fullpath_from_attr(dav_resource *res,
+		const char *p, const char *hex_chunkid)
+{
+	GError *err = NULL;
+	gchar xname[256], xvalue[1024];
```

Comment:
`LIMIT_LENGTH_CONTENTPATH` itself is 1024. `xvalue` should be `LIMIT_LENGTH_ACCOUNTNAME +  LIMIT_LENGTH_BASENAME + LIMIT_LENGTH_CONTENTPATH + LIMIT_LENGTH_VERSION + 32 + 4`. 32 is content ID, 4 is for separators.